code/sorting/src/Postman_Sort/Postman_Sort.c

```diff
@@ -0,0 +1,105 @@
+// C Program to Implement Postman Sort Algorithm
```

Comment:
Filename and folder name should be all lowercase.

Reply:
Should i add ` in postmans sort